tests/app/dao/test_services_dao.py

```diff
@@ -380,6 +384,34 @@ def test_get_all_user_services_should_return_empty_list_if_no_services_for_user(
     assert len(dao_fetch_all_services_by_user(user.id)) == 0
 
 
+@freeze_time('2019-04-23T10:00:00')
+def test_dao_fetch_live_services_data(sample_user, mock):
+    org = create_organisation()
+    service = create_service(go_live_user=sample_user)
+    template = create_template(service=service)
+    create_service(service_name='second', go_live_user=sample_user)
+    template2 = create_template(service=service, template_type='email')
+    dao_add_service_to_organisation(service=service, organisation_id=org.id)
+    create_ft_billing(bst_date='2019-04-20', notification_type='sms', template=template, service=service)
+    create_ft_billing(bst_date='2019-04-21', notification_type='sms', template=template, service=service)
+    create_ft_billing(bst_date='2018-04-20', notification_type='sms', template=template, service=service)
+    create_ft_billing(bst_date='2019-04-20', notification_type='email', template=template2,
```

Comment:
Should expand this test to have more than one service in ft_billing and letters

Reply:
ok 👍

Reply:
I added more data to the test now, so we have:
- one service with all kinds of records in ft_billing
- one service with letter records in ft_billing
- one service without ft billing records or go_live_user to see that it still appears